packages/core/admin/_internal/node/staticFiles.ts

```diff
@@ -17,6 +17,10 @@ const getEntryModule = (ctx: BuildContext): string => {
     .map(({ importName, path }) => `import ${importName} from '${path}';`)
     .join('\n');
 
+  const flags = {
+    contentReleases: process.env.FEATURE_FLAG_CONTENT_RELEASES
+  }
```

Comment:
how come you're passing this here? 👀

Reply:
I needed to pass this so I can use it [here. ](https://github.com/strapi/strapi/blob/c2480eb2649083db5a784fe1c9648d2de1250d85/packages/core/strapi/src/admin.ts) So I can conditionally add content-releases to the admin or not based on that feature flag.

Didn't ping you because is temporal for now, we need to discuss about this 🤔 so probably would change. But any suggestion is welcome!

Reply:
Can you not just use `process.env` directly in the file instead?

Reply:
I understand this function on `admin.ts` file is used in a client file

I tried that first but I have errors on my browser console because process is not defined on client of course

Reply:
> I tried that first but I have errors on my browser console because process is not defined on client of course

weird.

If you change this line – https://github.com/strapi/strapi/blob/116339ad8657ad2cf62135506bf64a1373295230/packages/core/admin/_internal/node/core/env.ts#L35

to be `key.toUpperCase().startsWith('STRAPI_ADMIN_') |  key.toUpperCase().startsWith('FEATURE_FLAG_')` does the issue still prevail?

Reply:
just to explain myself, this is highly exposed to users as an API and i'm not sure it really should be to be honest because it's not how we manage flags anywhere else and feels off :)

feel free to grab me if it's not working and we can investigate together.